### PR TITLE
add popup

### PR DESCRIPTION
**Pull request by AnnaGrynchuk**
### Popup

[Demo](https://annagrynchuk.github.io/Popup/) |
[Code base](https://github.com/AnnaGrynchuk/Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/AnnGrynchuk/popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
+
+body {
+    margin: 0 auto;
+    width: 100%;
+    height: 100%;
+}
+.wrapper {
+	display: flex;
```
Comment (A-Ostrovnyy) [inline submissions/AnnGrynchuk/popup/css/style.css]: Use prettier for formatting your code.

File: submissions/AnnGrynchuk/popup/index.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+                      <p>Vault</p>
+                    </a>
+                 </div> 
+                <label class="more_items" for="show_more" tabindex="50"></label> 
+          </div>
+      </div>
+  </div>
+  </body>
+</html>
+
```
Comment (A-Ostrovnyy) [inline submissions/AnnGrynchuk/popup/index.html]: REmove unused lines. Also, don't forget about an empty line at the end of every file.

**Comment by A-Ostrovnyy** [comment]
Good work!
Your icon menu without outline on `:focus` state. Also, I can iterate popups items it's good but I can't open the second part of the popup by pressing Space or Enter key on More button. 
Please fix it.

**Comment by AnnaGrynchuk** [comment]
@A-Ostrovnyy Thank you for review! I've tried to fix all.

File: submissions/AnnGrynchuk/popup/index.html [hunk 3]
```diff
@@ -1,149 +1,145 @@
...
-                 </div>    
-                 <div class="menu_item hidden" tabindex="16">
-                    <a href="#">
-                      <img class="icon" src="img/vault.png" alt="vault_icon">
-                      <p>Vault</p>
-                    </a>
-                 </div> 
-                <label class="more_items" for="show_more" tabindex="50"></label> 
-          </div>
+    <input class="popup_checkbox" id="popup" type="checkbox" tabindex="1">
```
Comment (A-Ostrovnyy) [inline submissions/AnnGrynchuk/popup/index.html]: `input type=checkbox` is already an interactive element and unnecessary add attribute `tabindex="1"`

**Comment by A-Ostrovnyy** [comment]
Good job!
When you added `tabindex` to `div.menu__item`, you added this element to the `:focus` context, but you have already had an interactive element: `<a>` inside of it. And when I am iterating popups item in the first round, I see the outline of `div.menu__item`.  If I continue iterating popups items by Tab key, I will see the outline of `<a>` tag.

**Comment by A-Ostrovnyy** [comment]
Great job!

**Pull request merged**